### ISSUE TITLE
luakit doesn't start up?

### ISSUE THREAD

**Issue by SolarAquarion** [luakit/luakit#316]
** (luakit:7211): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-ElPVSwGSNp: Connection refused

** (luakit:7211): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-ElPVSwGSNp: Connection refused
[    0.115376] I: luaH_loadrc:163: Loading rc: /etc/xdg/luakit/rc.lua
[    0.153049] E: luaH_dofunction:147: Lua error: module 'widget.uri' not found:
                        no field package.preload['widget.uri']                                                                                                                                
                        no file './widget/uri.lua'                                                                                                                                            
                        no file '/usr/share/luajit-2.0.5/widget/uri.lua'                                                                                                                      
                        no file '/usr/local/share/lua/5.1/widget/uri.lua'                                                                                                                     
                        no file '/usr/local/share/lua/5.1/widget/uri/init.lua'                                                                                                                
                        no file '/usr/share/lua/5.1/widget/uri.lua'                                                                                                                           
                        no file '/usr/share/lua/5.1/widget/uri/init.lua'                                                                                                                      
                        no file '/home/solaraquarion/.config/luakit/widget/uri.lua'                                                                                                           
                        no file '/home/solaraquarion/.config/luakit/widget/uri/init.lua'                                                                                                      
                        no file '/etc/xdg/luakit/widget/uri.lua'                                                                                                                              
                        no file '/etc/xdg/luakit/widget/uri/init.lua'                                                                                                                         
                        no file '/usr/share/luakit/lib/widget/uri.lua'                                                                                                                        
                        no file '/usr/share/luakit/lib/widget/uri/init.lua'                                                                                                                   
                        no file './widget/uri.so'                                                                                                                                             
                        no file '/usr/local/lib/lua/5.1/widget/uri.so'                                                                                                                        
                        no file '/usr/lib/lua/5.1/widget/uri.so'                                                                                                                              
                        no file '/usr/local/lib/lua/5.1/loadall.so'                                                                                                                           
                        no file './widget.so'                                                                                                                                                 
                        no file '/usr/local/lib/lua/5.1/widget.so'                                                                                                                            
                        no file '/usr/lib/lua/5.1/widget.so'                                                                                                                                  
                        no file '/usr/local/lib/lua/5.1/loadall.so'                                                                                                                           
                  Traceback:                                                                                                                                                                  
                  (1) [C]                       in function require                                                                                                                           
                  (2) /etc/xdg/luakit/rc.lua:44 in main chunk
[    0.153081] W: luaH_parserc:226: loading rc '/etc/xdg/luakit/rc.lua' failed

**Comment by groolot** [comment]
Is it from a fresh clone, from tar.gz?

Could you give more on the context?

**Comment by aidanholm** [comment]
Looks like your `/etc/xdg/luakit/rc.lua` is outdated for some reason. Try diffing that with the version on the `develop` branch.

**Comment by SolarAquarion** [comment]
@aidanholm i removed the "backups" in the PKGBUILD.  especially since the /etc/xdg/luakit is still unstable

**Comment by aidanholm** [comment]
Alright; did you manage to resolve this issue?

**Comment by SolarAquarion** [comment]
Yes
